src-cr/Components/Place.js

```diff
@@ -0,0 +1,161 @@
+import React from "react"
+
+import { Slide } from 'react-slideshow-image';
+
+import Event from "./Event"
+
+import Map from "./Map"
+
+import Food from "./Food"
+
+
+class Place extends React.Component {
+
+  state = {
+    place: [],
+  }
+
+  fetchPlace = () => (
+    fetch(`http://localhost:8000/places/${this.props.match.params.id}`)
+    .then(resp => resp.json())
+    .then(place => this.handleResponse(place))
+  )
+
+  componentDidMount = () => {
+    this.fetchPlace()
+  }
+
+  handleResponse = (place) => {
+    this.setState({
+      place: place
```

Comment:
When using ES6 as you're doing, you don't need to specify the key of an object if the const has the same name.
```JS
this.setState({ place }) // equals: { place: valueOfPlaceVariable }
```